packages/core/phase2/lib/generateOperations/generateOperations.test.ts

```diff
@@ -182,63 +182,6 @@ describe("generateOperations", () => {
     ])
   })
 
-  it("generates HO200121 exception when there are no recordable offences", () => {
```

Comment:
I think we need to have a test for when there are no recordable offences. Something like:
`it returns no operations when there are no recordable offences`